requirements.txt

```diff
@@ -1,4 +1,6 @@
 numpy>=1.22
 networkx>=2.8,<3.0
 z3-solver
-tensornetwork>=0.4
+quimb>=1.4.0
+autoray==0.6.0
```

Comment:
minor comment - is it necessary to be this specific version or do you think anything above 0.6.0 should work?

Reply:
Not necessary. The latest version of autoray is 0.6.0 right now, but it will probably work with versions above 0.6.0.

Reply:
I've updated requirements.txt